pkg/install/0-installstorage.go

```diff
@@ -37,7 +38,11 @@ var apiVersions = map[string]string{
 }
 
 func (i *Installer) createDNS(ctx context.Context) error {
-	return i.dns.Create(ctx, i.doc.OpenShiftCluster)
+	err := i.dns.Create(ctx, i.doc.OpenShiftCluster)
+	if err != nil {
```

Comment:
make `"recordset %q already registered"` an official error in the dns package and specifically test it here